Sources/Clappr/Classes/Base/Container.swift

```diff
@@ -63,6 +65,15 @@ open class Container: UIObject {
         }
     }
 
+    @objc open func reload(source: String) {
```

Comment:
Why would a `reload` receive another `source` instead of reloading the same received before?

Reply:
Fixed